### DISCUSSION TITLE
How to remove embedded PDF's from note?

### DISCUSSION THREAD

**Discussion by audioxntrk** [MicroPad/Web#451] [Q&A] [answered]
I have searched and searched for more in-depth documentation and cannot find out how to remove an embedded PDF.
Numerous posts mention the ability to embed them, but none seem to explain how to remove them.
You do not get the same window with the red delete button on embedded PDFs.

**Comment by NickGeek** [comment, marked as answer]
You can click on the bar (just not on the :::: itself because that moves it) above the PDF to go into editing mode on it.
